warehouse/warehouse/scripts/fix_mapfile_paths.py

```diff
@@ -1,4 +1,5 @@
 import sys
+import os
```

Comment:
since you switched to the pathlib method,  you should be able to remove this `os` import.

Reply:
Right.  I can do that and (remove os) and hopefully get Path correct.